docs/internals/contributing.rst

```diff
@@ -51,7 +51,7 @@ For Windows the use of cmd under Administrator permission is suggested instead o
 
 .. code-block:: batch
 
-    > virtualenv -p "C:\Python34\python.exe" env
+    > C:\Python34\python.exe -m venv env
```

Comment:
This assumes users will install on that location. This is not what windows10 offers as a default location.